docs/logging/docs/02-01-logging.md

```diff
@@ -2,8 +2,22 @@
 title: Architecture
 ---
 
-This document outlines the logging architecture of Kyma. It highlights information sources from which Logspout extracts logs to feed to OK Log.
+This document provides an overview of the logging architecture in Kyma. It describes the information sources from which promtail extracts logs to feed to Loki.
```

Comment:
```suggestion
This document provides an overview of the logging architecture in Kyma. 
```
I would remove the second part, since it's rather about the agent itself, log chunks and the read/write process - or we can add one line about them.